pkg/storage/segments/repairer.go

```diff
@@ -122,9 +142,15 @@ func (repairer *Repairer) Repair(ctx context.Context, path storj.Path) (err erro
 		return Error.Wrap(err)
 	}
 
+	requestCount := int(
+		math.Ceil(float64(redundancy.OptimalThreshold())*
```

Comment:
nit - pretty sure the cast to float64 is unnecessary now that the `multiplierOptimalThreshold` is a float already

Reply:
Also nit - I think this would be better split into multiple steps. One step to calculate `totalNeeded := math.Ceil(optimalThresh * multiplier)`, and another step to calculate `requestCount := totalNeeded - len(healthyPieces)`

Reply:
Ignore the comment about not casting to `float64`

Reply:
I will break the calculation in 2 steps but just using a scope for restricting the live of the `totalNeeded` variable as it's only needed for the calculating the `requestCount`